### PR TITLE
Card1.0

### PR DESCRIPTION
**Pull request by LoicBrassart**

### CODE REVIEW

**Review by LauraCourquin — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,27 @@
-import Home from "@pages/Home";
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: No blank line

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
 
+import Card from "@components/Cards/Card";
+// import Home from "@pages/Home";
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: No dead Code

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -1,12 +1,27 @@
...
+import LegalNotice from "./components/LegalNotice/LegalNotice";
+import Contact from "./components/Contact/Contact";
 
 function App() {
   return (
     <div className="App">
-      <Home />
-      <p>coucou</p>
+
+      {/* <Home /> */}
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: Dead code

File: frontend/src/components/Carousel/index.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  const [images, setImages] = useState(CarouselData);
+  const [position, setPosition] = useState(0);
+
+  useEffect(() => {
+    axios.get("http://localhost:5000/heroes").then(({ data }) => {
+      setImages(data);
+    });
+  }, []);
+  const prevSlide = () => {
+    // déterminez si position est égal à 0 et si l'utilisateur a atteint le début du carrousel.
```
Comment (LuigiMenez) [inline frontend/src/components/Carousel/index.jsx:16]: Commentary in English please

File: frontend/src/pages/Home.jsx [hunk 5]
```diff
@@ -1,36 +1,10 @@
...
 
-export default function Home() {
-  return (
-    <header className="App-header">
-      <img src={logo} className="App-logo" alt="logo" />
-      <p>Hello Vite + React !</p>
+// import Card from "@components/Card";
 
-      <Counter />
+// export default function Home() {
```
Comment (LuigiMenez) [inline frontend/src/pages/Home.jsx]: Dead Code

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Warning, Dead code, Blank Line, and the commentary in English please

**Review by LauraCourquin — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok

**Pull request merged**